src/utils/currencies.ts

```diff
@@ -4,11 +4,20 @@ export enum Currency {
   Euro = 'cEUR',
 }
 
+// Important: when adding new currencies, the string must match the symbol
+// we use in address-metadata
 export enum CiCoCurrency {
   CELO = 'CELO',
-  CUSD = 'CUSD',
-  CEUR = 'CEUR',
-  CREAL = 'CREAL',
+  cUSD = 'cUSD',
+  cEUR = 'cEUR',
+  cREAL = 'cREAL',
+}
+
+export const cicoCurrencyTranslationKeys = {
```

Comment:
nit: does this need to be here? Or can this just be closer to the front end component?

Reply:
ty for catching that. At one point i used it in multiple files, but that closed away